src/hooks/useAppBootstrap.functions.ts

```diff
@@ -23,14 +24,18 @@ export const parseOnboardingURL = (
 
     const onboardingUrl = new URL(url)
     const onboardUrl = onboardingUrl.searchParams.get('onboard_url')
+    const onboardedRedirection = onboardingUrl.searchParams.get(
+      'onboarded_redirection'
+    )
     const fqdn = onboardingUrl.searchParams.get('fqdn')
 
-    if (!onboardUrl && !fqdn) {
+    if (!onboardUrl && !onboardedRedirection && !fqdn) {
```

Comment:
I'm not sure this check should be added. If we don't have `onboardUrl` nor `fqdn`, then there would be no sense to have an `onboardedRedirection`.

If the deep link contains only `onboardedRedirection` but no `onboardUrl` nor `fqdn`, then the app would break as it would try to initiate onboarding without enough params.

So I would return `undefined` in that case (so I would keep the old check)

Reply:
100% correct.